internal/provider/api/models.go

```diff
@@ -1,12 +1,20 @@
 package api
 
+type ClusterParameters struct {
+	Type   string
+	Region string
+	Cloud  string
+}
+
 type VirtualCluster struct {
 	ID            string `json:"id"`
 	Name          string `json:"name"`
 	Type          string `json:"type"`
 	AgentPoolID   string `json:"agent_pool_id"`
 	AgentPoolName string `json:"agent_pool_name"`
 	CreatedAt     string `json:"created_at"`
+	Region        string `json:"virtual_cluster_region"`
+	CloudProvider string `json:"virtual_cluster_cloud_provider"`
```

Comment:
@aratz-lasa you want to change those fields in the `/describe_` payload to be `region` and `cloud_provider`, right?